### ISSUE TITLE
error on importing.

### ISSUE THREAD

**Issue by ahmedtehseen** [dmtrKovalenko/material-ui-datetimepicker#9]
`import DateTimePicker from 'material-ui-datetimepicker';`
I'm getting this error after just importing it.
![error](https://user-images.githubusercontent.com/13465129/31397665-0aac2f32-ae00-11e7-89cc-501d0513b83d.png)

**Comment by dmtrKovalenko** [comment]
Oh, there is no css in this package, this looks weird.
Try to update to 1.0.0 if you are using react 16.0.0

**Comment by ahmedtehseen** [comment]
Thanks @dmtrKovalenko it works.
But there are couple of warnings shows up please take a look.
![warnings](https://user-images.githubusercontent.com/13465129/31430703-16eaeda2-ae8b-11e7-9f76-43a94554c2d2.JPG)

I did exactly as mentioned in Docs.
```
import DatePickerDialog from 'material-ui/DatePicker/DatePickerDialog';
import TimePickerDialog from 'material-ui/TimePicker/TimePickerDialog';

<DateTimePicker 
  onChange={this.setDate} 
  DatePicker={DatePickerDialog}
  TimePicker={TimePickerDialog}
/>
```

**Comment by dmtrKovalenko** [comment]
:sunglasses: Fixed in [v1.0.1](https://github.com/dmtrKovalenko/material-ui-datetimepicker/tree/1.0.1)

**Comment by ahmedtehseen** [comment]
@dmtrKovalenko Thanks mate all warnings have gone.
